e2e-tests/gef/cypress/e2e/clone/clone-gef-deal.spec.js

```diff
@@ -111,7 +111,6 @@ context('Clone GEF (AIN) deal', () => {
 
         applicationDetails.bankRefName().contains('Cloned AIN deal');
         applicationDetails.automaticCoverStatus().contains('Not started');
-        applicationDetails.facilityStatus().contains('Completed');
```

Comment:
The facility status wont be `Completed` since the FED values are removed (when cloning to version 0) - It also assumes that the cover start date wasnt removed too (which will happen if it was in the past)

Reply:
Thanks - can we replace with the expected status i.e `IN_PROGRESS`?